components/locale.js

```diff
@@ -61,14 +61,26 @@ function getInitialLocaleProps({req} = {}) {
   return Promise.resolve({locale})
 }
 
-// eslint-disable-next-line complexity
 function getContent(locale, options) {
+  const fallbackContent = content('', options)
+  if (locale === fallbackLocale) {
+    return fallbackContent
+  }
+  const localeContent = content(`${locale}/`, options)
+  return Object.keys(fallbackContent).reduce((cont, key) => {
```

Comment:
Actually, you know what, we require the filename to be provided for these files, so I think we could associate it that way!